### PR TITLE
3.2.0 release post

### PR DESCRIPTION
**Pull request by parkr**
@benbalter @pathawks @jekyll/core – this look good to you?

### CODE REVIEW

**Comment by pathawks** [comment]
**Wow!** What an impressive list of enhancements in this release :+1:

LGTM :ship:

**Comment by DirtyF** [comment]
@parkr I'm mentioned twice in the contributor list, that's way too much compared to what major contributors have achieved, could you please remove `DirtyF` from the list?

**Comment by parkr** [comment]
> @parkr I'm mentioned twice in the contributor list, that's way too much compared to what major contributors have achieved, could you please remove DirtyF from the list?

Yes, will do. It's generated from the output of `git log` so if you ever commit with `DirtyF` instead of `Frank Taillandier` then it shows up as such. 😄

**Comment by envygeeks** [comment]
LGTM. :shipit:

File: site/_posts/2016-07-26-jekyll-3-2-0-released.markdown [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+- Ankush Menat
+- Anthony Smith
+- Ben Balter
+- Brian Jones
+- Brint O'Hearn
+- Chayoung You
+- Chris Wells
+- chrisfinazzo
+- Clark Winkelmann
+- crisp
```
Comment (crispgm) [inline site/_posts/2016-07-26-jekyll-3-2-0-released.markdown]: Both crisp & David Zhang are me. Remove the crisp?

**Comment by benbalter** [comment]
Nice work team. LGTM.

**Pull request merged**